ts/components/wallet/TransactionsList.tsx

```diff
@@ -84,7 +78,7 @@ export default class TransactionsList extends React.Component<Props> {
       <Row>
         <Left>
           <Text>
-            {isNew && <IconFont name="io-new" style={styles.newIconStyle} />}
+            {isNew && <BadgeComponent />}
```

Comment:
to avoid this
![Schermata 2019-08-07 alle 12 30 08](https://user-images.githubusercontent.com/822471/62616738-9744e200-b910-11e9-8795-3f2e63ffa86b.png)

I changed this
```typescript
<Text>
   {isNew && <BadgeComponent />}
   <Text note={true}>{isNew ? `  ${datetime}` : datetime}</Text>
</Text>
```

with this
```typescript
<View style={{ flexDirection: "row", alignItems: "center" }}>
      {isNew && <BadgeComponent />}
      <Text note={true}>{isNew ? `  ${datetime}` : datetime}</Text>
</View>
```
To get this
![Schermata 2019-08-07 alle 12 39 16](https://user-images.githubusercontent.com/822471/62616843-c4919000-b910-11e9-9bbb-f2b742e09c67.png)

Reply:
Yes, in this way it is aligned